flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/InfluxdbReporterTest.java

```diff
@@ -111,7 +111,7 @@ public void testMetricReporting() throws Exception {
 			reporter.report();
 
 			verify(postRequestedFor(urlPathEqualTo("/write"))
-				.withQueryParam("db", equalTo(TEST_INFLUXDB_DB))
+				.withQueryParam(InfluxdbReporterOptions.DB.key(), equalTo(TEST_INFLUXDB_DB))
```

Comment:
Please revert this change here, since the query param is actually unrelated to how we name our options. In my last review, I was only referring to the `createMetricRegistry` method below.